src/logic/map_objects/tribes/warelist.h

```diff
@@ -61,7 +62,7 @@ struct WareList {
 	}
 
 private:
-	std::vector<Quantity> wares_;
+	std::vector<std::atomic<Quantity>> wares_;
```

Comment:
this change caused the compiler errors, so I assume doing it this way does not met the atomic criteria

Reply:
Sorry, it looks like I didn't compile it on my system...